Automattic/woocommerce-payments
# Updated Link by Stripe logo

Fixes #8366

#### Changes proposed in this Pull Request

This PR updates the Link by Stripe logo according to the discussion on the original issue #8366.

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/0bbcb478-96b0-44ce-9a30-3388c1dcc0ae)

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/2ad5ed3a-0dee-4d39-8883-832c01f1bb03)


#### Testing instructions





1. Enable Link by Stripe in WooPayments settings. While on it, ensure the logo is updated there.
2. Check Blocks Checkout and Shortcode Checkout and ensure the new logo is being displayed and it works as expected.

-------------------


**Post merge**
